spec/src/main/asciidoc/core/spi.asciidoc

```diff
@@ -1693,7 +1693,7 @@ These method should never be invoked upon bean instances. Otherwise, unpredictab
 * `configure()` returns an `AnnotatedTypeConfigurator` (as defined in <<annotated_type_configurator>>) initialized with the instance type to easily apply specific interceptor binding to apply when enhancing the instance.
 The method always return the same `AnnotatedTypeConfigurator`
 * `createInterceptedInstance()` returns an enhanced version of the instance for which each method invocations will be a business method invocation.
```

Comment:
May require follow up - does this imply the self-method calls limitation is lifted?